x-pack/plugins/triggers_actions_ui/common/data/lib/build_agg.test.ts

```diff
@@ -793,6 +793,51 @@ describe('buildAgg', () => {
       });
     });
 
+    it('should create correct aggregation when condition params are defined and multi terms selected', async () => {
```

Comment:
if you remove the fix and run this test, you'll see the same `TypeError: Cannot set properties of undefined (setting 'order')` error that we see in the logs